### ISSUE TITLE
Error sending simple mail

### ISSUE THREAD

**Issue by asherber** [typesafe/mnailgun#23]
The following code is throwing an exception: `RuntimeBinderException: 'Newtonsoft.Json.Linq.JObject' does not contain a definition for 'id'`. The same params work if I hit the API with RestSharp. Am I missing something here? 

Using .NET 4.6.2, mnailgun 1.0.0.

Thanks.

```csharp
var domain = "sandboxxxx.mailgun.org";
var key = "key-328xxx";
var client = new MailgunClient(domain, key, 3);
var msg = new MailMessage()
{
	From = new MailAddress("mailgun@samples.mailgun.org", "Excited User"),
	Subject = "Test of library",
	Body = "This is a test of mnailgun."
};
msg.To.Add("me@gmail.com");  // note: using real address here
client.SendMail(msg);
```

**Comment by asherber** [comment]
Some more information: I was running the above code in LinqPad. Despite the exception; the mail still went out; the exception was thrown in parsing the response.

When I created an actual VS console app, I got instead a file load error in JSON.Net. Poking around a little, I found that mnailgun was pulling in JSON.Net 3.5.8, very far from the current 9.0.1. Even updating JSON.Net to 5.0.1 worked just fine. I would suggest that you might want to update the minimum required JSON.Net version in your NuGet package.
